### ISSUE TITLE
ClassCastException in bindImageView function in javafx

### ISSUE THREAD

**Issue by bivsbon** [jzy3d/jzy3d-api#178]
**Describe the bug**
Function bindImageView() throws exception: class org.jzy3d.chart.controllers.mouse.camera.AWTCameraMouseController cannot be cast to class org.jzy3d.javafx.controllers.mouse.JavaFXCameraMouseController

**Screenshots**
![image](https://user-images.githubusercontent.com/31033536/119094953-dee02380-ba3b-11eb-8b91-6c88187ecafb.png)

**Reproducibility**
Run the code in jzy3d-api/jzy3d-native-jogl-javafx/src/main/java/org/jzy3d/demos/javafx/DemoJzy3dFX.java

**Versions (please complete the following information):**
 - JVM / JDK: 16

**Additional context**

**Comment by jzy3d** [comment]
Which version of Jzy3d are you using?
The master branch is currently on 2.0.0-SNAPSHOT and will be soon released. Support for JavaFX has been suspended since [Oracle does not plan to support JavaFX in the future](https://www.oracle.com/technetwork/java/javase/javaclientroadmapupdate2018mar-4414431.pdf) (and JavaFX is not anymore in the JDK).  

If you are not tied to JavaFX, I suggest you use AWT or Swing charts.

**Comment by bivsbon** [comment]
I'm using Jzy3d 2.0.0-SNAPSHOT version, and I'm doing a college project that requires JavaFX so can't switch to other UI tool.

**Comment by jzy3d** [comment]
Then try 1.0.3 or 1.0.2 that where compatible with JavaFX

**Comment by bivsbon** [comment]
Yeah, I'll stick to the previous version, seems to be working just fine
